### ISSUE TITLE
[Itemdb] Baby Dragon Hat (5531)

### ISSUE THREAD

**Issue by Niunzin** [HerculesWS/Hercules#1506]
The item has no bonus. (DEX +2 and Lv. 5 Double Attack)

Need to add:
Script: <" bonus bDex,2; skill TF_DOUBLE,5; bonus bDoubleRate,5; ">

**Comment by Mysteries** [comment]
Is this hat even supposed to have a bonus? 

Divine - http://www.divine-pride.net/database/item/5531/baby-dragon-hat-1
iRO Pre - http://db.irowiki.org/olddb/item-info/5531/
iRO RE - http://db.irowiki.org/db/item-info/5531/

**Comment by Asheraf** [comment]
Yep kRO Raw has a bonus description (RE).

**Comment by Mysteries** [comment]
Ty

**Comment by iRagno** [comment]
According to kRO, this hat has:

> 머리 위에서 잠이 든 귀여운 아기 용.
> 이대로 무얼하든 깨어나지 않을 것 같다.
> DEX+2
> 5레벨 더블어택 사용 가능.
> 더 높은 레벨을 배우고 있을 시, 해당 레벨로 발동.
> 계열 : 투구 방어 : 1
> 위치 : 상단 무게 : 10
> 요구 레벨 : 1
> 장착 : 전 직업

Translates to (Google Translator) :

> For a cute baby sleeping on the head.
> No matter what I do, I will not wake up.
> DEX + 2
> Level 5 Double Attack available.
> If you are learning a higher level, activate it at that level.
> Category: Helm Defense: 1
> Location: Top Weight: 10
> Requires Level: 1
> Mounting: former occupation

Description is also the same than Chick_Hat, except for Double Attack skill level.

Source: http://ro.gnjoy.com/guide/runemidgarts/itemview.asp?itemSeq=2&ID=5531

**Comment by iRagno** [comment]
I have talked about this with @Ridley8819. According with issue tracker http://herc.ws/board/tracker/issue-5935-sidewinder-card-effect, items that enables Double Attack at certain level activates Double Attack effect on every weapon and also activates Thief's Double Attack skill icon on Skill Window.

According with that, the script for Baby Dragon should be:
```
	Script: <"
		bonus bDex, 2;
		skill TF_DOUBLE, 5;
		bonus bDoubleRate, 25;
 	">
```

**Comment by dastgirp** [comment]
Fixed in #1620 ( https://github.com/HerculesWS/Hercules/commit/d25481239d1c873c9395696a25690a0174df5ede )
